app/components/Tooltip.tsx

```diff
@@ -35,6 +37,7 @@ function Tooltip({ shortcut, content: tooltip, delay = 50, ...rest }: Props) {
       animation="shift-away"
```

Comment:
Can these props be removed since the source tooltip in `TooltipContext` has them?
Maybe have just the `delay` supported here to allow override?